modules/combined/test/tests/frictional_contact/single_point_2d/sm/tests

```diff
@@ -0,0 +1,28 @@
+[Tests]
+  issues = '#6524'
+  design = 'syntax/Contact/index.md MechanicalContactConstraint.md'
+  [./single_point_2d_sm]
+    type = 'Exodiff'
+    input = 'single_point_2d_sm.i'
+    exodiff = 'single_point_2d_sm_out.e'
+    custom_cmp = 'single_point_2d.cmp'
+    superlu = true
+    requirement = 'The contact system shall enforce 2D single-point contact with significant accumulated slip. Legacy Solid Mechanics.'
+  [../]
+  [./single_point_2d_predictor_sm]
+    type = 'Exodiff'
+    input = 'single_point_2d_predictor_sm.i'
+    exodiff = 'single_point_2d_predictor_sm_out.e'
+    custom_cmp = 'single_point_2d.cmp'
+    superlu = true
+    requirement = 'The contact system shall enforce 2D single-point contact with significant accumulated slip. With predictor solver options. Legacy Solid Mechanics. '
+  [../]
+  [./single_point_2d_tp_sm]
+    type = 'Exodiff'
+    input = 'single_point_2d_tp_sm.i'
+    exodiff = 'single_point_2d_tp_sm_out.e'
+    custom_cmp = 'single_point_2d.cmp'
+    superlu = true
+    requirement = 'The contact system shall enforce 2D single-point contact with significant accumulated slip when formulation selected is penalty contact. Legacy Solid Mechanics. '
```

Comment:
penalty -> tangential_penalty
Also, remove the extra whitespace at the end of this requirement and the one above.